aepsych/server/server.py

```diff
@@ -409,12 +423,12 @@ def handle_setup(self, request):
             experiment_id = None
             if self._db_master_record is not None:
                 experiment_id = self._db_master_record.experiment_id
-
+            
             self._db_master_record = self.db.record_setup(
-                description="default description",
```

Comment:
I don't have an opinion one way or the other about capitalization, but do make sure that it's consistent. It might be useful to define `DEFAULT_NAME` and `DEFAULT_DESC` constants.

Reply:
Good idea